domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -123,6 +130,23 @@ class TopicListController @Inject constructor(
   }
 
   private fun createTopicList(): TopicList {
+    return if (loadLessonProtosFromAssets) {
+      val topicIdList =
+        assetRepository.loadProtoFromLocalAssets(
+          assetName = "topics",
```

Comment:
Here this kind of breaks the paradigm for assetName compared to the fields that are being used above, so I think it may be better to keep assetType, assetId, and baseMessage

Reply:
It's not a type though, it's the actual file name checked into the APK. Calling it 'assetType' would be misleading. Perhaps 'assetBaseFIilename' would be the most clear, but it's a bit lengthy. I also don't understand how you're distinguishing between 'type' and 'ID'. This is the topic list so it doesn't have an ID.

It could be named better but I'm following existing filenames to keep things a bit simpler. We'll be removing all of this code in coming months, anyway, in favor of downloading & caching completely dynamically.

Reply:
Same comment as above..given that this code will be dead soon I think its okay

Reply:
Ack, thanks.